### PR TITLE
nfs-rgw unsupported - add new images and note

### PR DESCRIPTION
**Pull request by asettle**
Images to come

### CODE REVIEW

**Comment by asettle** [comment]
Screenshot from 2020-10-23 14-07-29
This is the new note (without screenshots)

File: xml/admin_gui_dashboard.xml [hunk 1]
```diff
@@ -2324,9 +2324,13 @@ EOF
     </step>
     <step>
      <para>
-      Select a storage back-end&mdash;either <guimenu>CephFS</guimenu> or
-      <guimenu>Object Gateway</guimenu>.
+      Select a storage back-end.
      </para>
+      <important>
+        <para>
+          At this time, &ganesha; exports backed by the &ogw; are unsupported.
```
Comment (asettle) [inline xml/admin_gui_dashboard.xml:2331]: This depends: we could also say "At this time, &ganesha; exports backed by &cephfs; are the only supported option." or something. @lproven what do we like?

@LenzGr @mgfritch

Comment (LenzGr) [reply xml/admin_gui_dashboard.xml:2331]: How about "At this time, only NFS exports backed by CephFS are supported."

**Review by LenzGr — APPROVED** [review]
LGTM, thanks!

File: xml/admin_gui_dashboard.xml (hunk 1 above)
Comment (lproven) [reply xml/admin_gui_dashboard.xml:2331]: I like @LenzGr's suggestion here. Anything else I can come up with is wordier.

**Review by lproven — CHANGES_REQUESTED** [review]

**Review by mgfritch — APPROVED** [review]

**Review by tbazant — APPROVED** [review]
good

**Pull request merged**